book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+class Book
+  attr_accessor :label, :author
+  attr_reader :rentals
```

Comment:
- [ ] One of the instance variables is missing inside the setters and getters initialization. 

please add the `: title` to the attr_accessor and you can even add the `: rentals` to the attr_accessor to set has many rentals.